docs/mkdocs.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+site_name: Gossamer Docs
+site_description: The Official Documentation for Gossamer | A Go Implementation of the Polkadot Host
+
+repo_name: chainsafe/gossamer
+repo_url: https://github.com/ChainSafe/gossamer
+
+# Configuration
+markdown_extensions:
+  - meta
+  - admonition
+  - codehilite
+  - toc:
+      permalink: true
+
+theme:
+  name: material
+  palette:
+    primary: pink
+    accent: indigo
+  favicon: ./assets/favicon.ico
+  features:
+    # - navigation.instant
+    - navigation.tracking
+    - navigation.sections
+    - navigation.tabs
+    - navigation.tabs.sticky
+nav:
+
+  - Getting Started:
+    - Overview: ./index.md
+    - Host Architecture: ./getting-started/overview/host-architecture.md
+    - Package Library: ./getting-started/overview/package-library.md
+    - Installation: ./getting-started/installation.md
+    - General Resources: ./getting-started/resources/general-resources.md
+    - Developer Resources: ./getting-started/resources/developer-resources.md
+  - Usage:
```

Comment:
Can we add a section titled "Running Nodes" and just copy over the three sections outlining how to run nodes from the Installation section?